app/src/main/java/com/amaze/filemanager/database/daos/BookmarkEntryDao.java

```diff
@@ -47,7 +48,7 @@
 @Dao
 public interface BookmarkEntryDao {
 
-  @Insert
+  @Insert(onConflict = OnConflictStrategy.IGNORE)
```

Comment:
Will this ignore every insertion after the first one?

Reply:
Would it be sensible if the same record having the same name and path get overwritten again and again, but it won't change the position of the bookmark at the drawer itself?